prinz_pmml/src/main/scala/pl/touk/nussknacker/prinz/pmml/repository/PMMLModelPayload.scala

```diff
@@ -0,0 +1,13 @@
+package pl.touk.nussknacker.prinz.pmml.repository
+
+import java.io.InputStream
+
+case class PMMLModelPayload(inputStream: InputStream, name: String, version: String)
```

Comment:
imo we should  use the case classes as final (as Maciek mentioned that there are problems with inheritance for them and it's not useful to have inheritance here)